### PR TITLE
Improve visibility of gun bolts

### PR DESCRIPTION
**Pull request by notquitehadouken**
## About the PR

Lets the player know if the gun is bolted by showing a red circle with a line through it on their cursor
Makes the text that shows up when you try to fire an unbolted gun show up on the cursor

## Why / Balance

Because it's hard to notice that your gun's bolt is open if there are no visual indicators aside from a small bit of the sprite looking different. Also because people are usually looking at their cursor when trying to fire a gun.

## Technical details

Adds a new texture for the reticle around the cursor

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/space-wizards/space-station-14/assets/45953835/78b99a52-334d-4aa4-858e-bef56e149376

## Breaking changes

**Changelog**

:cl:
- tweak: The reticle around your cursor will change depending on if the gun you're holding is bolted or not.
- tweak: The "Gun not bolted" text that shows up whenever you try to fire a gun with an open bolt now shows up on your cursor instead of your character.

### CODE REVIEW

**Comment by DogZeroX** [comment]
The text is repeating!

**Comment by notquitehadouken** [comment]
> The text is repeating!

visibility :trollface:

File: Content.Client/CombatMode/CombatModeIndicatorsOverlay.cs [hunk 1]
```diff
@@ -1,9 +1,11 @@
 using System.Numerics;
 using Content.Client.Hands.Systems;
 using Content.Shared.Weapons.Ranged.Components;
+using FastAccessors;
```
Comment (metalgearsloth) [inline Content.Client/CombatMode/CombatModeIndicatorsOverlay.cs:4]: ???

Comment (notquitehadouken) [reply Content.Client/CombatMode/CombatModeIndicatorsOverlay.cs:4]: yeah i dunno

**Comment by metalgearsloth** [comment]
@notquitehadouken popup still needs fixing please.

**Comment by notquitehadouken** [comment]
> @notquitehadouken popup still needs fixing please.

will do when i figure out how

**Pull request merged**